workflow/config/config.go

```diff
@@ -64,6 +64,10 @@ type WorkflowControllerConfig struct {
 
 	// Config customized Docker Sock path
 	DockerSockPath string `json:"dockerSockPath,omitempty"`
+
+	// Workflow defaults
+	//[TODO] fix the representation
+	WorkflowDefaults *wfv1.WorkflowSpec `json:"workflowDefaults,omitempty"`
```

Comment:
I think we should call this `defautWorkflowSpec`.

Reply:
Should it be capital "d" to keep it consistent? ```DefautWorkflowSpec```